.github/workflows/ci.yml

```diff
@@ -31,6 +31,14 @@ env:
   CARGO_INCREMENTAL: 0
 
 jobs:
+  check:
+    name: check
+    runs-on: ubuntu
+    steps:
+      - name: check
+        run: |
+          echo "node input: ${{ inputs.node_version }}"
```

Comment:
I assume you'll add something in this job?

Reply:
This was just to check, but it's removed now